src/Components/Facility/DoctorCapacityForm.tsx

```diff
@@ -232,99 +226,83 @@ export const DoctorCapacityForm = (props: DoctorCapacityProps) => {
               handleSubmit(e);
             }}
           >
-            <CardContent>
-              <InputLabel
-                htmlFor="area-of-specialization"
-                id="demo-simple-select-outlined-label"
-              >
-                Area of specialization*
-              </InputLabel>
-              <NativeSelectField
+            <div className="p-4">
+              <FieldLabel className="mb-2" required={true}>
+                Area of specialization
+              </FieldLabel>
+              <SelectMenuV2
                 id="area-of-specialization"
-                name="area"
-                variant="outlined"
-                value={state.form.area}
-                options={doctorTypes}
-                onChange={handleChange}
+                value={doctorTypes.find((type) => type.id == state.form.area)}
+                options={doctorTypes.filter((type) => !type.disabled)}
+                optionLabel={(option) => option.text}
+                onChange={(e) =>
+                  handleChange({ name: "area", value: (e && e.id) || "" })
+                }
                 disabled={!!id}
               />
               <ErrorHelperText error={state.errors.area} />
-            </CardContent>
-            <CardContent>
-              <InputLabel
-                htmlFor="count"
-                id="demo-simple-select-outlined-label"
-              >
-                Count*
-              </InputLabel>
-              <TextInputField
+            </div>
+            <div className="p-4">
+              <FieldLabel htmlFor="count" required={true} className="mb-2">
+                Count
+              </FieldLabel>
+              <TextFormField
                 id="count"
                 name="count"
-                variant="outlined"
-                margin="dense"
                 type="number"
-                InputLabelProps={{ shrink: !!state.form.count }}
                 value={state.form.count}
-                onChange={handleChange}
-                errors={state.errors.count}
+                onChange={(e) =>
+                  handleChange({ name: "count", value: e.value })
+                }
+                error={state.errors.count}
               />
-            </CardContent>
-            <CardContent>
-              <Grid container spacing={2} className="flex justify-between">
-                <Grid item className="flex flex-row w-full sm:w-auto gap-4">
-                  <Button
+            </div>
+            <div className="p-4">
+              <div className="flex justify-between flex-col md:flex-row">
+                <div className="flex flex-row w-full sm:w-auto gap-4">
+                  <ButtonV2
                     id="doctor-cancel"
+                    variant="secondary"
                     className="w-full sm:w-auto"
-                    color="default"
-                    variant="contained"
                     onClick={() => goBack(!id && `/facility/${facilityId}`)}
```

Comment:
As these buttons are inside a form tag, clicking on them would submit the form by default, so use event.preventDefault() or type="button"

```suggestion
                    onClick={(e) => {
                        e.preventDefault()
                        goBack(!id && `/facility/${facilityId}`);
                    }}
```